libs/controller/controller.py

```diff
@@ -678,7 +704,7 @@ def __init__(self, credentials=None):
         self.branch = credentials["branch"]
         ssl._create_default_https_context = ssl._create_unverified_context
 
-    def get_latest_build(self, model=None):
+    def get_latest_build(self, model=None, version=None):
```

Comment:
This method should be renamed to get_build. That is the intended logic we want to build. Correct?

Reply:
done!